src/components/structures/auth/ForgotPassword.tsx

```diff
@@ -83,6 +86,10 @@ interface State {
 }
 
 export default class ForgotPassword extends React.Component<Props, State> {
+    public static defaultProps: Partial<Props> = {
+        onLoginClick: () => {},
```

Comment:
Why is `onLoginClick` not a required prop?

Reply:
Don't know thb. But every place using it passes `onLoginClick` as a prop → Made it required